lib/jekyll/site.rb

```diff
@@ -279,9 +279,10 @@ def site_payload
     # Get the implementation class for the given Converter.
     # Returns the Converter instance implementing the given Converter.
     # klass - The Class of the Converter to fetch.
-
     def find_converter_instance(klass)
-      converters.find { |klass_| klass_.instance_of?(klass) } || \
+      @find_converter_instance ||= {}
+      @find_converter_instance[klass] = \
+        converters.find { |klass_| klass_.instance_of?(klass) } || \
```

Comment:
I realise this didn't change in your PR but why is the block variable called `klass_` instead of e.g. `converter` ? Seems a bit odd to me.

Reply:
Agreed. I can change it since it's part of the diff..